modules/cql/src/blaze/elm/compiler/clinical_values.clj

```diff
@@ -72,8 +73,14 @@
 
 
 ;; 3.6. Concept
-;;
-;; TODO
+(defn- compile-codes [context codes]
+  (map #(core/compile* context %) codes))
+
+(defmethod core/compile* :elm.compiler.type/concept
+  [context
+   {:keys [codes]}]
+  (let [codes_comp (compile-codes context codes)]
```

Comment:
I would remove the `let` here, because everything fits on one line and the binding name doesn't add much clarity because the function name `compile-codes` says all.